File: fish_JS_answer/MochaChai/Extra/Fish.js
class Fish {
	constructor (theFishOwner, newCost, newColor, newBreed, newGender) {
		this.myFishOwner = theFishOwner
		this.cost = newCost
		this.color = newColor
		this.breed = newBreed
		this.gender = newGender
	}

	toString () {
		let result = ''
		result +=  "    " + this.color + " " + this.breed + " (" + this.gender + ") worth $" + this.cost.toString()
		return result
	}
}

File: fish_Python_answer/fish.py
class Fish:
    def __init__(self, new_cost, new_color, new_breed, new_gender, the_owner):
        self.my_owner = the_owner
        self.cost = new_cost
        self.color = new_color
        self.breed = new_breed
        self.gender = new_gender

    def __str__(self):
        return f'{self.color} {self.breed} [{self.gender}] worth ${self.cost}'
